content/applications/general/export_import_data.rst

```diff
@@ -176,156 +228,171 @@ Example that will not work:
-``purchase.order_functional_error_line_cant_adpat.CSV`` file of some quotations you can import,
-based on demo data.
+If a company wants to import a sales order with several order lines, a specific row must be reserved
+in the :abbr:`CSV (Comma-separated Values)` file for each order line. The first order line will be
+imported on the same row as the information relative to order. Any additional lines will need an
+additional row that does not have any information in the fields relative to the order.
 
-:download:`File for some Quotations <export_import_data/purchase.order_functional_error_line_cant_adpat.csv>`.
+As an example, here is :abbr:`CSV (Comma-separated Values)` file of some quotations that can be
+imported, based on demo data:
 
-The following CSV file shows how to import purchase orders with their respective purchase order
-lines:
+- :download:`File for some Quotations
+  <export_import_data/purchase.order_functional_error_line_cant_adpat.csv>`.
 
-:download:`Purchase orders with their respective purchase order lines <export_import_data/o2m_purchase_order_lines.csv>`.
+The following :abbr:`CSV (Comma-separated Values)` file shows how to import purchase orders with
+their respective purchase order lines:
 
-The following CSV file shows how to import customers and their respective contacts:
+- :download:`Purchase orders with their respective purchase order lines
+  <export_import_data/o2m_purchase_order_lines.csv>`.
 
-:download:`Customers and their respective contacts <export_import_data/o2m_customers_contacts.csv>`.
+The following :abbr:`CSV (Comma-separated Values)` file shows how to import customers and their
+respective contacts:
 
-Can I import several times the same record?
--------------------------------------------
+- :download:`Customers and their respective contacts
+  <export_import_data/o2m_customers_contacts.csv>`.
 
-If you import a file that contains one of the column "External ID" or "Database ID", records that
-have already been imported will be modified instead of being created. This is very useful as it
-allows you to import several times the same CSV file while having made some changes in between two
-imports. Odoo will take care of creating or modifying each record depending if it's new or not.
+Can the same record be imported several times?
+----------------------------------------------
 
-This feature allows you to use the Import/Export tool of Odoo to modify a batch of records in your
-favorite spreadsheet application.
+If a file is imported that contains one of the columns: `External ID` or `Database ID`, records that
+have already been imported will be modified instead of being created. This is extremely useful as it
+allows to import the same :abbr:`CSV (Comma-separated Values)` file several times while having made
+some changes in between two imports. Odoo will take care of creating or modifying each record
+depending if it is new or not.
```

Comment:
```suggestion
If an imported file contains one of the columns: :guilabel:`External ID` or :guilabel:`Database ID`,
records that have already been imported are modified, instead of being created. This is extremely
useful as it allows users to import the same :abbr:`CSV (Comma-separated Values)` file several
times, while having made some changes in between two imports. 

Odoo takes care of creating or modifying each record, depending if it is new or not.
```